varya/assets/sass/blocks/layout-grid/_style.scss

```diff
@@ -0,0 +1,17 @@
+@import 'config';
+
+.wp-block-jetpack-layout-grid {
+    grid-gap: var(--layout-grid--gutter-large) !important;
+
+    @each $size in $layout-gutter-sizes {
+        &.wp-block-jetpack-layout-gutter__#{$size} {
+            grid-gap: var(--layout-grid--gutter-#{size}) !important;
```

Comment:
Yup, as you pointed out, let’s work out another way to solve this without `!important` as the last step before merging.